frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -126,8 +126,10 @@ class OverrideForkPointDialog(
               isSelected: Boolean,
               cellHasFocus: Boolean
             ): Component {
-              val commit: ICommitOfManagedBranch = value as ICommitOfManagedBranch
```

Comment:
Here's the actual fix for #1436

Reply:
how does the dialog look like when the commit is null?

Reply:
Huh good point

<img width="643" alt="image" src="https://user-images.githubusercontent.com/3383210/211912381-201b42f8-0aaa-4d7f-b876-ba1a7ac38f4a.png">

So the entire `Inferred commit` item should be hidden then... lemme fix

Reply:
And `commit` param can probably only be null when `val list` is empty, lemme check... 🤔

Reply:
Ha! nope, `commit` will be `null` many times, as `getListCellRendererComponent` for some reasons gets invoked for `index = -1` and `value = null`

Reply:
Okay, all related issues fixed